datadog/dogstatsd/base.py

```diff
@@ -160,10 +173,12 @@ def __enter__(self):
 
         def __exit__(self, type, value, traceback):
             # Report the elapsed time of the context manager.
-            self.statsd.timing(self.metric, time() - self.start,
-                               self.tags, self.sample_rate)
+            elapsed = time() - self.start
+            use_ms = self.statsd.use_ms or self.use_ms
```

Comment:
Thanks @clokep, this is, indeed, unexpected. 

I think having this property is worth: it makes convenient to switch specific timers from `s` to `ms` without altering the overall `statsd` configuration.